runtime/doc/api-functions.txt

```diff
@@ -0,0 +1,722 @@
+
+nvim_tabpage_list_wins({tabpage})                   *nvim_tabpage_list_wins()*
+                Gets the windows in a tabpage
+
+                Parameters:~
+                    {tabpage}  Tabpage
+
+                Return:~
+                    List of windows in tabpage
+
+nvim_tabpage_get_var({tabpage}, {name})               *nvim_tabpage_get_var()*
+                Gets a tab-scoped (t:) variable
+
+                Parameters:~
+                    {tabpage}  Tabpage handle
+                    {name}     Variable name
+
+                Return:~
+                    Variable value
+
+nvim_tabpage_set_var({tabpage}, {name}, {value})      *nvim_tabpage_set_var()*
+                Sets a tab-scoped (t:) variable
+
+                Parameters:~
+                    {tabpage}  Tabpage handle
+                    {name}     Variable name
+                    {value}    Variable value
+
+nvim_tabpage_del_var({tabpage}, {name})               *nvim_tabpage_del_var()*
+                Removes a tab-scoped (t:) variable
+
+                Parameters:~
+                    {tabpage}  Tabpage handle
+                    {name}     Variable name
+
+nvim_tabpage_get_win({tabpage})                       *nvim_tabpage_get_win()*
+                Gets the current window in a tabpage
+
+                Parameters:~
+                    {tabpage}  Tabpage handle
+
+                Return:~
+                    Window handle
+
+nvim_tabpage_get_number({tabpage})                 *nvim_tabpage_get_number()*
+                Gets the tabpage number
+
+                Parameters:~
+                    {tabpage}  Tabpage handle
+
+                Return:~
+                    Tabpage number
+
+nvim_tabpage_is_valid({tabpage})                     *nvim_tabpage_is_valid()*
+                Checks if a tabpage is valid
+
+                Parameters:~
+                    {tabpage}  Tabpage handle
+
+                Return:~
+                    true if the tabpage is valid, false otherwise
+
+
+==============================================================================
+3. UI Functions                                                  *nvim-api-ui*
+
+remote_ui_disconnect({channel_id})                    *remote_ui_disconnect()*
+                TODO: Documentation
+
+                                                            *nvim_ui_attach()*
+nvim_ui_attach({channel_id}, {width}, {height}, {options})
+                TODO: Documentation
+
+nvim_ui_detach({channel_id})                                *nvim_ui_detach()*
+                TODO: Documentation
+
+nvim_ui_try_resize({channel_id}, {width}, {height})     *nvim_ui_try_resize()*
+                TODO: Documentation
+
+nvim_ui_set_option({channel_id}, {name}, {value})       *nvim_ui_set_option()*
+                TODO: Documentation
+
+
+==============================================================================
+4. Vim Functions                                                *nvim-api-vim*
```

Comment:
Is "Vim" coming from the filename, `vim.c`? It's probably time that we renamed it to `core.c` or else special-cased it. We've moved away from the "vim" prefix and it's mostly just confusing at this point.